airflow/cli/commands/config.py

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Config sub-commands"""
+import io
+
+import pygments
+from pygments.lexers.configs import IniLexer
+import rich_click as click
+
+from airflow.configuration import conf
+from airflow.utils.cli import should_use_colors
+from airflow.utils.code_utils import get_terminal_formatter
+from airflow.cli import airflow_cmd, click_color
+
+click_section = click.argument(
+    'section',
+)
+
+click_option = click.argument(
+    'option',
+)
+
+@airflow_cmd.group()
+@click.pass_context
+def config(ctx):
+    """Commands for the metadata database"""
+
+@config.command('list')
+@click.pass_context
+@click_color
+def show_config(ctx, color):
+    """Show current application configuration"""
+    with io.StringIO() as output:
+        conf.write(output)
+        code = output.getvalue()
+        if should_use_colors(color):
+            code = pygments.highlight(code=code, formatter=get_terminal_formatter(), lexer=IniLexer())
+        print(code)
+
+
+@config.command('get-value')
+@click.pass_context
+@click_section
+@click_option
+def get_value(ctx, section, option ):
+    """Get one value from configuration"""
+    if not conf.has_section(section):
+        raise SystemExit(f'The section [{section}] is not found in config.')
+
+    if not conf.has_option(section, option):
+        raise SystemExit(f'The option [{section}/{option}] is not found in config.')
+
+    value = conf.get(section, option)
+    print(value)
```

Comment:
Let's use `rich.console.Console` to print things.

Reply:
<img width="517" alt="Screenshot 2022-06-17 at 9 06 47 PM" src="https://user-images.githubusercontent.com/10162465/174331042-56317d0e-7b97-45b1-b61d-9bf979d36144.png">
This is the print of the config without rich console
<img width="517" alt="Screenshot 2022-06-17 at 9 07 11 PM" src="https://user-images.githubusercontent.com/10162465/174331124-b204754b-93aa-4772-a3fc-05a9de7aaba2.png">
This is the print of the config with rich console. The section of the configuration is getting removed if I use rich console. I will see how to fix this issue. 
@blag